components/frontend/src/subject/SubjectTitle.js

```diff
@@ -45,6 +46,15 @@ function SubjectTypeAndName({ subject, subject_uuid, subject_name, reload }) {
                     />
```

Comment:
Maybe rename this component to "configuration" instead "SubjectTypeAndName" as it is now more than just the type and the name? (on line 27 of this file, where, weirdly, I can not place a comment)

Reply:
Yes, good suggestion.